flang/test/Lower/OpenMP/omp-wsloop-chunks.f90

```diff
@@ -0,0 +1,97 @@
+! This test checks lowering of OpenMP DO Directive(Worksharing).
```

Comment:
NIT: Can you make this comment a bit more specific ?

Reply:
Updated.